Remotion/ObjectBinding/Web/Services/BocAutoCompleteReferenceValueSearchResult.cs

```diff
@@ -0,0 +1,39 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+//
+namespace Remotion.ObjectBinding.Web.Services
+{
+  /// <summary>
+  /// The base search result of a <see cref="IBocAutoCompleteReferenceValueWebService.Search"/> call.
+  /// </summary>
+  public abstract class BocAutoCompleteReferenceValueSearchResult
+  {
+    /// <summary>
+    /// Creates a new <see cref="BocAutoCompleteReferenceValueSearchResultWithValueList"/> instance, containing the given values.
```

Comment:
```suggestion
    /// Creates a new <see cref="BocAutoCompleteReferenceValueSearchResult"/> based on the <paramref name="values"/>.
```

Since we're hiding the specific return type anyway, there's no need to put it into the summary. Using the paramref covers doccing the parameter for "simple" doc scenarios and adds a nice touch.

Also, after seeing this in the docs, the paramname feels better as `values` instead of `proxies`.